drud/ddev
# Turn off strict ssh host key checking in container

## The Problem/Issue/Bug:

Composer commands were being interrupted if the host key had not yet been accepted. We should probably not require that in the context of our web container.

## How this PR Solves The Problem:

Add +StrictHostKeyChecking=no

## Manual Testing Instructions:

* `docker pull drud/ddev-webserver:v1.4.0`
* Use `ddev auth ssh` to authenticate
* `ddev ssh` and `rm ~/.ssh-agent/known-hosts` (To delete the shared known_hosts file)
* In the container, ssh to a server that should accept your key; you should get in without accepting the host key.

## Automated Testing Overview:


## Related Issue Link(s):

## Release/Deployment notes:
